### ISSUE TITLE
Pagination and Screen Reader(s)

### ISSUE THREAD

**Issue by mtribone** [projectblacklight/blacklight#1906]
When testing Blacklight 7 with the available online demo (http://demo.projectblacklight.org/) our users with the accessibility group experienced some difficulty with the previous/next and numbered pagination.

1. Previous and next links are hard to use. It doesn't tell the user that there is a change when using the links, meaning that they've gone to another section of results, and this was confusing and unexpected.
1. Numbered pagination doesn't announce where you go or are in the results when using them to navigate between the pages of results. Again give the user better audible feedback of actions in regards to where they are in the search results list.

**Comment by barmintor** [comment]
Cf #1905

**Comment by mejackreed** [comment]
Perhaps also related to https://github.com/projectblacklight/blacklight/pull/1970 ? @mtribone does that PR get at what you are looking for?

**Comment by mtribone** [comment]
@mejackreed Yeah I think that might do it. We'd have to test again with our user, but we can always open another issue if it doesn't quite get it all.
